### ISSUE TITLE
No milestones selectable in first new issue (but after it's ok)

### ISSUE THREAD

**Issue by TeChn4K** [k41n/redmine_milestones#13]
Process :
Create version
Create milestones for this version
Create issue 1 => unable to chose milestones but version OK
Create issue 2 => Chose version and chose milestone

(Redmine 1.4.1.stable.9690 (MySQL))
Great plugin ! Thanks a lot !

**Comment by TeChn4K** [comment]
Some more details:

When i create new issue and milestones isn't selectable after chosen version : refresh page and everything returns to normal. Weird ...

**Comment by lokisite** [comment]
Same issue.  Create an issue, select a version, no milestone option available.  Edit the issue and milestone is now available as a choice.  I think some clarification on useability would be helpful here -- should Milestone be an option only after a Version is selected?  And if so, should Milestone appear when the version is selected, or is one supposed to re-edit the issue in order to adjust milestone?     Redmine  2.1.2.

**Comment by bkleef** [comment]
Same here.
